### PR TITLE
SVG representation for Linear Vesting

### PR DESCRIPTION
**Pull request by 0xJem**
Completes #168

### CODE REVIEW

**Comment by 0xJem** [comment]
@Oighty I noticed that you branched off `master`. Should we rebase off `develop` instead?

File: src/modules/derivatives/LinearVesting.sol [hunk 1]
```diff
@@ -42,6 +43,7 @@ contract LinearVesting is DerivativeModule, ILinearVesting {
     // ========== MODULE SETUP ========== //
 
     constructor(address parent_) Module(parent_) {
+        // LinearVestingCard()
```
[the remark this answers is not in the document]
Comment (Oighty) [reply src/modules/derivatives/LinearVesting.sol:46]: Yes, I removed it when I was trying to fix compile issues. Will add back.

File: src/modules/derivatives/LinearVesting.sol [hunk 2]
```diff
@@ -682,4 +684,42 @@ contract LinearVesting is DerivativeModule, ILinearVesting {
...
+    /// @dev        This function reverts if:
+    ///             - The token ID does not exist
+    function tokenURI(uint256 tokenId_)
+        public
+        view
+        override
+        onlyValidTokenId(tokenId_)
+        returns (string memory)
+    {
+        Token storage token = tokenMetadata[tokenId_];
```
[the remark this answers is not in the document]
Comment (Oighty) [reply src/modules/derivatives/LinearVesting.sol:700]: I think the `onlyValidTokenId` check accounts for that, right?

File: src/modules/derivatives/LinearVestingCard.sol [hunk 3]
```diff
@@ -0,0 +1,553 @@
...
+        'font-family="\'Menlo\', monospace" fill="white" text-anchor="middle"';
+    string internal constant NULL = "";
+    string internal ADDR_STRING;
+
+    Colors internal colors;
+
+    // ========== CONSTRUCTOR ========== //
+
+    constructor() {
+        colors = Colors({
```
[the remark this answers is not in the document]
Comment (Oighty) [reply src/modules/derivatives/LinearVestingCard.sol:41]: Yeah, might be more efficient to do it that way so it doesn't need to be initialized. Originally did it the other way bc the API is a bit cleaner.

File: utils/hot-chain-svg/package.json
[the remark this answers is not in the document]
Comment (Oighty) [reply utils/hot-chain-svg/package.json:1]: It's in it's own folder. Since it's a javascript utility, I didn't want to mingle it with the solidity files.

**Comment by 0xJem** [comment]
@Oighty should I merge after reviewing?

**Comment by Oighty** [comment]
> @Oighty should I merge after reviewing? (and fixing the tests - probably salts)

Yes, this should be good now once the CI is fixed.

**Pull request merged**